### PR TITLE
[RW-6010][risk = no] remove more vestiges of old svc

### PR DESCRIPTION
**Pull request by jaycarlton**
Remove dead config option and all functions for dealing with `QueryParameterValue` objects in the upload pipeline. Should have been part of https://github.com/all-of-us/workbench/pull/4355

@calbach FYI
---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI + API server with `yarn test-local`
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

**Comment by jmthibault79** [comment]
please lint and push

File: api/config/config_local.json [hunk 1]
```diff
@@ -123,7 +123,6 @@
   },
   "reporting": {
     "dataset": "reporting_local",
-    "maxRowsPerInsert": 200,
-    "uploadMethod": "STREAMING"
```
Comment (jmthibault79) [inline api/config/config_local.json:127]: When was `uploadMethod` last used in the code?  Please ensure that there is at least one release between that merge and this one.

File: api/src/main/java/org/pmiops/workbench/reporting/insertion/InsertAllRequestPayloadTransformer.java [hunk 2]
```diff
@@ -27,11 +28,19 @@
  * @code{ InsertAllRequestPayloadTransformer<ReportingUser> userTransformer =
  *     UserColumnValueExtractor::values; }
  */
+@FunctionalInterface
 public interface InsertAllRequestPayloadTransformer<MODEL_T>
     extends BigQueryInsertionPayloadTransformer<MODEL_T> {
   String INSERT_ID_CHARS = "abcdefghijklmnopqrstuvwxyz";
   int INSERT_ID_LENGTH = 16;
+  int MAX_ROWS_PER_INSERT_ALL_REQUEST = 1000;
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/reporting/insertion/InsertAllRequestPayloadTransformer.java:36]: Should this use the `maxRowsPerInsert` config value?

**Review by jmthibault79 — COMMENTED** [review]
Looks good - a few requested changes.

cron_default.yaml still refers to `reporting.uploadMethod` - Please update that comment

File: api/src/main/java/org/pmiops/workbench/reporting/insertion/InsertAllRequestPayloadTransformer.java (hunk 2 above)
Comment (jaycarlton) [reply api/src/main/java/org/pmiops/workbench/reporting/insertion/InsertAllRequestPayloadTransformer.java:36]: It does, at least in the new code. The constant is a cap on the configured value.

File: api/config/config_local.json (hunk 1 above)
Comment (jaycarlton) [reply api/config/config_local.json:127]: 🔪  https://github.com/all-of-us/workbench/pull/4355/files#diff-fd748e8599d99102a8da44afd79c7e9483084d1857c5bf983e4252a9b5243087L66

Comment (jaycarlton) [reply api/config/config_local.json:127]: Good point. I'll leave this in place for rollback safety.

File: api/src/main/java/org/pmiops/workbench/reporting/insertion/InsertAllRequestPayloadTransformer.java [hunk 3]
```diff
@@ -27,11 +28,19 @@
...
 
+  default List<InsertAllRequest> buildBatchedRequests(
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/reporting/insertion/InsertAllRequestPayloadTransformer.java:38]: This does not seem to be used anywhere?

Comment (jaycarlton) [reply api/src/main/java/org/pmiops/workbench/reporting/insertion/InsertAllRequestPayloadTransformer.java:38]: It's new code from a different branch. Sholuldn't be here...

**Review by jmthibault79 — APPROVED** [review]

**Pull request merged**